ReactWindows/ReactNative.Shared/ReactInstanceManager.cs

```diff
@@ -467,7 +467,7 @@ private Task<ReactContext> CreateReactContextCoreAsync(CancellationToken token)
 
         private Task<ReactContext> CreateReactContextFromDevManagerAsync(CancellationToken token)
         {
-            if (_devSupportManager.HasUpToDateBundleInCache())
+            if (!_devSupportManager.IsRemoteDebuggingEnabled && _devSupportManager.HasUpToDateBundleInCache())
```

Comment:
I put the `!_devSupportManager.IsRemoteDebuggingEnabled` check here because `CreateReactContextFromCachedPackagerBundleAsync` doesn't seem to work with the WebSocket executor. @rozele what are your thoughts on the right fix? Should `CreateReactContextFromCachedPackagerBundleAsync` and the WebSocket executor work together? If so, what work is required to do that?

Reply:
Can you just move the `IsRemoteDebuggingEnabled` check into the `HasUpToDateBundleInCache` check?

---
In reply to: [174006430](https://github.com/Microsoft/react-native-windows/pull/1713#discussion_r174006430) [](ancestors = 174006430)

Reply:
Technically speaking, if you are remote debugging then you don't have the right bundle cached :-)

---
In reply to: [174761436](https://github.com/Microsoft/react-native-windows/pull/1713#discussion_r174761436) [](ancestors = 174761436,174006430)

Reply:
Basically, we can treat "HasUpToDateBundleInCache" as "ShouldRunFromCachedBundle"

---
In reply to: [174761518](https://github.com/Microsoft/react-native-windows/pull/1713#discussion_r174761518) [](ancestors = 174761518,174761436,174006430)